docs/addon/retry.md

```diff
@@ -0,0 +1,131 @@
+---
+id: retry
+---
+
+# Retry Addon
+
+Retry addon for [Fiber](https://github.com/gofiber/fiber) designed to apply retry mechanism for unsuccessful network
+operations. This addon uses an exponential backoff algorithm with jitter. It calls the function multiple times and tries
+to make it successful. If all calls are failed, then, it returns an error. It adds a jitter at each retry step because adding
+a jitter is a way to break synchronization across the client and avoid collision.
+
+## Table of Contents
+
+- [Retry Addon](#retry-addon)
+- [Table of Contents](#table-of-contents)
+- [Signatures](#signatures)
+- [Examples](#examples)
+- [Default Config](#default-config)
+- [Custom Config](#custom-config)
+- [Config](#config)
+- [Default Config Example](#default-config-example)
```

Comment:
The sections are using `##` but this references `#`

Reply:
this is ok

its the anchor feature and the other one is the headline hashtag
the headline number e.g. ### test -> is at the end #test as an anchor